src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -673,6 +673,21 @@ def removeTask(self, taskName):
         self.data.tasks.tasklist.remove(taskName)
         return
 
+    def getSiteWhitelist(self):
+        """
+        Get the site white list for the workflow
+        :return: site white list
+        """
+        if getattr(self.data, 'SiteWhiteList', []):
```

Comment:
Looking into the setter implementation, it looks like this if statement will never evaluate to true. Did you also observe it in the unit tests?

Reply:
removed